Bottom: 81e2fda531d9b691d4d1f5927e5d89a4c57de8b6
Top:    36145a4ced59be217f2a3b7bebbbd8ee76f523bb
Author: Alexey Lyashkov <shadow@Alexeys-MacBook-Pro.local>
Date:   2016-01-21 10:46:56 +0300

Refresh of ctl-verbs.patch

---

diff --git a/ib-sock-ctl.c b/ib-sock-ctl.c
index 4f35dcd..16b76ce 100644
--- a/ib-sock-ctl.c
+++ b/ib-sock-ctl.c
@@ -42,6 +42,14 @@ ib_sock_ctl_take(struct IB_SOCK *sock)
  */
 int ib_sock_ctl_post(struct IB_SOCK *sock, struct ib_sock_ctl *msg)
 {
+	/* base on ib_mad_post_receive_mads() - that wr isn't need to live any time 
+	 * so - save memory and declare on stack.
+	 */
+	struct ib_recv_wr wr;
+	struct ib_recv_wr *bad_wr;
+
+	ret = ib_post_recv(rx->qp, &wr, &bad_wr);
+
 	return 0;
 }
